go.sum

```diff
@@ -1,23 +1,43 @@
 github.com/BurntSushi/toml v0.3.1/go.mod h1:xHWCNGjB5oqiDr8zfno3MHue2Ht5sIBksp03qcyfWMU=
```

Comment:
Can you run `go mod tidy`, and commit the changes, please? So that we only have the required dependencies, and we don't commit any dependency that is actually not needed.

Reply:
Ran go mod tidy and updated version in the latest commit